package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "geojson-vt",
-  "version": "3.2.1",
+  "version": "3.2.2",
```

Comment:
Can you remove this change? I prefer to do it as a part of a separate commit (automatically made with `npm version`) when releasing a new version.

Reply:
Oh shoot, my bad. Habits die hard.